app/views/homepage/_hero_image_and_search_form.html.erb

```diff
@@ -17,7 +17,8 @@
                 <div class="digital-description col-12 col-md-6 col-lg-12">
                   <div class="pre-heading">Science History Institute</div>
                     <h2 class="brand-alt-h2">Digital Collections</h2>
-                    <p>Search <%= number_with_delimiter(Work.count) %> of our digitized items: artifacts, photographs, advertisements, letters, rare books, and more. You can also learn more about the rest of our <a href="https://sciencehistory.org/collections">collections.</a></p>
+                    <% published_item_count = Kithe::Model.where(:published => true, :type => ['Work', 'Collection']).count %>
+                    <p>Search <%= number_with_delimiter(published_item_count) %> of our digitized items: artifacts, photographs, advertisements, letters, rare books, and more. You can also learn more about the rest of our <a href="https://sciencehistory.org/collections">collections.</a></p>
```

Comment:
In an idea world we wouldn't put local variable declarations in our ERB templates. (As part of a general desire to keep complex logic out of ERB altogether, but local variable declarations seem particularly galling). 

Part of the problem with Rails is that it lacks a good place to pull this logic out to (which we solve with our "ViewModels", but we aren't using that here, and in general haven't ever used one for a top-level template rather than a partial). 

The standard Railss answer is to put them in "helper methods", but these end up global and are terrible. 

Another thing you can do is put them in the *controller*, with a `helper_method` declaration in the controller to make it available to the view. So a method `published_item_count` maybe. 

Or, in this case, i think it would be fine to just shove it inline `number_with_delimiter(Kithe::Model.where(:published => true, :type => ['Work', 'Collection']).count)` -- not great, but still better than a local variable in ERB I think, and good enough.

Reply:
Cool. Inline seems simpler - it's a simple enough calculation.